core/src/test/java/ai/timefold/solver/core/api/solver/SolutionManagerTest.java

```diff
@@ -17,9 +17,6 @@
 import ai.timefold.solver.core.impl.testdata.domain.TestdataEntity;
 import ai.timefold.solver.core.impl.testdata.domain.TestdataSolution;
 import ai.timefold.solver.core.impl.testdata.domain.TestdataValue;
-import ai.timefold.solver.core.impl.testdata.domain.allows_unassigned.TestdataAllowsUnassignedEntity;
```

Comment:
Thinking out loud:

- Package names here are too long, we ought to shorten.
- We have two kinds of testdata - domains and utils.

So what if we:

- Move domains to `ai.timefold.solver.core.testdomain`.
- Move utils to `ai.timefold.solver.core.testutil`.

Notice how I dropped the "impl". That's 5 characters less, and no significant loss of information.
I also dropped "data", another 5 characters less.

Reply:
Or maybe... thinking out of the box here... the domains can move to something like `org.acme.testdomain`?

Reply:
I liked the idea of using `org.acme.testdomain`.

Reply:
What about `ai.timefold.solver.core.domain` and `ai.timefold.solver.core.util`?

Reply:
It is already a test resource. Do we really need the prefix `test`?

Reply:
Good question. Not sure.

The idea of `testdata` was to have a package which doesn't and couldn't possibly map to anything in the `src/main` of the Solver. `core.test` arguably could (and it does in the `solver-test` module), `core.testdomain` is unlikely to.

`org.acme` removes that problem altogether, but I'm not sure I like it better. It's for sure the shortest.

Reply:
I changed my mind regarding `org.acme.testdomain` and I am unsure about using a completely different pattern.